src/main/java/pro/sky/courseworktelegrambot/entities/StateButtonPK.java

```diff
@@ -0,0 +1,24 @@
+package pro.sky.courseworktelegrambot.entities;
+
+import java.io.Serializable;
+
+public class StateButtonPK implements Serializable {
+        private String state_id;
```

Comment:
```suggestion
        private String stateId;
```
camelCase, а не snake_case